src/index.ts

```diff
@@ -5,10 +5,21 @@ import { createApp } from './server'
 // Configuration
 const BEE_API_URL = process.env.BEE_API_URL || 'http://localhost:1633'
 const AUTH_SECRET = process.env.AUTH_SECRET
+const POSTAGE_STAMP = process.env.POSTAGE_STAMP
+const POSTAGE_DEPTH = process.env.POSTAGE_DEPTH
+const POSTAGE_AMOUNT = process.env.POSTAGE_DEPTH
+const POSTAGE_USAGE = Number(process.env.POSTAGE_USAGE || 0.7)
```

Comment:
I am not sure what these variables should represent or what goal you are following with having those here? 

I would expect that you provide only the Stamp Batch Id and if needed all of these parameters can always be fetched directly from the Bee node and it seems bit pointless to specify those manually as part of input to the server... Also the parameters could change during runtime (calling topup/dilute changes depth/amount and `usage` I guess changes on its own)

Reply:
So this is for the phase 3 of the project the auto-buying postage stamps. I just wanted to have the types ready on the interface so that it is clear why the interface is so "complicated". These are unused for now.

Reply:
The aim is that user would be able to define the `DEPTH` and `AMOUNT` and the algorithm will check if there is suitable stamp and if not buy it. It will also autobuy new one when some `USAGE` is reached

Reply:
I understand that and it would be alright for me if I could see how you plan to use those, which I don't understand.

Is that meant to be the defaults values for new batches? That would make sense for "amount" and "depth". Then "utilization" is meant as threshold? If so I would use different naming for those.

Reply:
Yes exactly. Do you want me to write a comment about it or is it fine to have them described in readme when i implement it (tomorrow).